theme/gatsby-node.js

```diff
@@ -68,11 +82,23 @@ function getEditUrl(repo, filePath) {
   return `https://github.com/${repo.user}/${repo.project}/edit/master/${filePath}`
 }
 
-async function fetchContributors(repo, filePath) {
+async function fetchContributors(repo, filePath, accessToken) {
+  const hash = `${repo.user}/${repo.project}/${filePath}`
+  const cached = CONTRIBUTOR_CACHE.get(hash)
+  if (cached) {
+    console.debug('found something in the cache. whee!')
```

Comment:
[Minor] I'd say we should use sentence case for consistency.

```suggestion
    console.debug('Found something in the cache. Whee!')
```

Reply:
Oh, looks like you took care of this already 👍